acslib/ccure/connection.py

```diff
@@ -32,18 +32,18 @@ def __init__(self, **kwargs):
 
     @property
     def base_headers(self):
-        """."""
+        """Headers required for each request to CCure"""
         return {
             "session-id": self.get_session_id(),
             "Access-Control-Expose-Headers": "session-id",
         }
 
     @property
-    def header_for_form_data(self):
+    def HEADER_FOR_FORM_DATA(self):
```

Comment:
I've never seen this before, a class property is still a function or method in python so should be lowercase. It isn't a constant. Why the upper case?

Reply:
Yeah, that's a good point. I made the change because the property returns a hard-coded constant value, and it's used in the code as a constant, eg `headers=self.connection.base_headers | self.connection.HEADER_FOR_FORM_DATA`.
But you're right, it doesn't seem to be conventional to use caps for a `@property` method